CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## Unreleased
+
+#### Added
+- Added support for [Contentful Delivery API Search Parameters](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters)
+- Swift 3.1, Xcode 8.3 support.
+
+#### Changed
+- 
+
+#### Removed
+- 
+
+#### Fixed
+- 
+
+#### Deprecated
+-
+
+---
+
+## Table of contents
+
+#### 0.x Releases
+- `0.3.x` Releases - [0.3.0](#030) | [0.3.1](#031)
+
+---
+
+## [`0.3.1`](https://github.com/contentful/contentful.swift/releases/tag/0.3.1)
+Released on 2017-02-03.
+
+#### Added
+- Support for installation via Carthage on all of iOS, macOS, tvOS, & watchOS.
+
+---
+
+## [`0.3.0`](https://github.com/contentful/contentful.swift/releases/tag/0.3.0)
+Released 2017-01-08.
+
+#### Added
+- **Breaking: Swift 3, Xcode 8**
```

Comment:
IMHO not a useful comment. I'd phrase it like `Converted to use Swift 3 and Xcode 8. Older versions are not supported.`